### PR TITLE
docs: added elixir-prql to book

### PR DESCRIPTION
**Pull request by kasvith**
- Added elixir bindings to the book

### CODE REVIEW

File: book/src/bindings/elixir.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# PRQL
```
Comment (max-sixty) [inline book/src/bindings/elixir.md]: If this is just the readme, let's include with `{{ include` like in https://github.com/PRQL/prql/blob/main/book/src/language-features/standard-library.md?plain=1#L20, and then it'll stay updated?

Comment (kasvith) [reply book/src/bindings/elixir.md]: aa wait

Comment (kasvith) [reply book/src/bindings/elixir.md]: @max-sixty can you do a build on docs? i cant do it in my PC due to the python error i mentioned before

Comment (max-sixty) [reply book/src/bindings/elixir.md]: Yeah, I just labeled it so it'll deploy to the main docs, easy to undo if it looks bad

Comment (kasvith) [reply book/src/bindings/elixir.md]: https://github.com/PyO3/pyo3/issues/2731 this issue keeps hitting me :(

Comment (kasvith) [reply book/src/bindings/elixir.md]: image

Comment (max-sixty) [reply book/src/bindings/elixir.md]: You have a 32bit python interpreter?! :)

Comment (kasvith) [reply book/src/bindings/elixir.md]: Yeah, need to reinstall but im bit lazy :D

**Comment by max-sixty** [comment]
Thanks @kasvith !

**Comment by kasvith** [comment]
Shall we do this for all bindings?

**Comment by max-sixty** [comment]
> Shall we do this for all bindings?

Yes that's a good idea!

BTW we can also add prql-elixir here, maybe when we publish to Hex? 
image

**Pull request merged**